src/user/supla_esp_devconn.c

```diff
@@ -149,12 +215,133 @@ devconn_smooth smooth[SMOOTH_MAX_COUNT];
 	#define supla_espconn_connect espconn_secure_connect
 #endif
 
+#if defined(POWSENSOR2)
+void DEVCONN_ICACHE_FLASH supla_get_voltage();
+void DEVCONN_ICACHE_FLASH supla_get_current();
+void DEVCONN_ICACHE_FLASH supla_get_power();
+void ICACHE_FLASH_ATTR sel_pin_voltage();
+void DEVCONN_ICACHE_FLASH supla_esp_em_extendedvalue_to_value(TElectricityMeter_ExtendedValue *ev, char *value);
+
+_supla_int_t SRPC_ICACHE_FLASH srpc_ds_async_channel_extendedvalue_changed(
+    void *_srpc, unsigned char channel_number,
+    TSuplaChannelExtendedValue *value);
+_supla_int_t SRPC_ICACHE_FLASH srpc_evtool_v1_emextended2extended(
+    TElectricityMeter_ExtendedValue *em_ev, TSuplaChannelExtendedValue *ev);
+_supla_int_t SRPC_ICACHE_FLASH srpc_evtool_v1_extended2emextended(
+    TSuplaChannelExtendedValue *ev, TElectricityMeter_ExtendedValue *em_ev);
+#endif
 
 void DEVCONN_ICACHE_FLASH supla_esp_devconn_timer1_cb(void *timer_arg);
 void DEVCONN_ICACHE_FLASH supla_esp_wifi_check_status(void);
 void DEVCONN_ICACHE_FLASH supla_esp_devconn_iterate(void *timer_arg);
 void DEVCONN_ICACHE_FLASH supla_esp_devconn_reconnect(void);
 
+#ifndef _RTC_H_
```

Comment:
It is not necessary to fetch the time from an external server. In this case, it is completely unnecessary and takes up unnecessary memory, which is not enough. Remove it.
For the future if you realy need... https://github.com/SUPLA/supla-espressif-esp/blob/develop/supla-common/srpc.h#L175